index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Hello Microverse!</h1>
```

Comment:
Great job @osamaashraf6 🤝

- [ ] [Optional] As stated in the project requirements, kindly add a class to this element and then apply styles to this element through this created class.